code/game/objects/items/weapons/tools.dm

```diff
@@ -273,6 +273,24 @@
 		else
 			user << "<span class='notice'>You are already trying to pull out a teeth!</span>"
 		return
+	if(ishuman(C) && C.dna.species.id == "wire" && C != user)
+		var/mob/living/carbon/human/H = C
+		H.visible_message("<span class='danger'>[user] begins to snip [H] into a bunch of little pieces!</span>", \
+							"<span class='userdanger'>[user] starts to snip you into a bunch of little pieces!</span>")
+		if(!do_mob(user, H, 150))
+			return
+		H.emote("scream")
+		for(var/obj/item/bodypart/L in H.bodyparts)

```

Comment:
Please use _istypeless for loops_ you can find more information about this in the [contributing.md](https://github.com/HippieStation/HippieStation13/blob/master/CONTRIBUTING.md)